staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -156,6 +156,10 @@ func testWebhookConverter(t *testing.T, watchCache bool) {
 		},
 	}
 
+	// Toggle "KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE" to allow for 1.32 flowcontrol v1beta3 API removal to not block tests
```

Comment:
I think we'd leave this here indefinitely so future APIs keep being testable as long as they can be served. We'd remove the v1beta3 flowcontrol fixtures from etcd data as part of removing serving code, not this bit

Reply:
Same comment in other integration tests

Reply:
I've updated this comment to reflect the suggestion above, it now removes mention of v1beta3 flowcontrol and being temporary.  It now reads:

```
	// KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE allows for APIs pending removal to not block tests
```